tests/cli/test_worker.py

```diff
@@ -533,6 +551,32 @@ async def test_install_policy_never(self, monkeypatch, prefect_client):
 
         run_process_mock.assert_not_called()
 
+        @pytest.mark.parametrize("worker_type", ["prefect-agent"])
+        def test_start_with_prefect_agent_type(worker_type):
+            # Run the CLI command with the specified worker_type
+            result = subprocess.run(
+                ["python", "worker.py", "worker", "start", "--type", worker_type],
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                text=True,
+            )
+
+            if worker_type == "prefect-agent":
+                assert result.returncode == 1
+
+                assert (
+                    "prefect-agent' typed work pools work with Prefect Agents"
+                    " instead of"
+                    in result.stderr
+                )
+            else:
+                assert result.returncode != 1
+                assert (
+                    "prefect-agent' typed work pools work with Prefect Agents"
+                    " instead of"
+                    not in result.stderr
+                )
```

Comment:
This test looks good! But I would recommend matching the style of the existing tests for this module. That would look like this:
```suggestion
        def test_start_with_prefect_agent_type(worker_type):
            invoke_and_assert(
                command=[
                    "worker",
                    "start",
                    "--run-once",
                    "-p",
                    "test-work-pool",
                    "-n",
                    "test-worker",
                    "-t",
                    "prefect-agent",
                ],
                expected_code=1,
                expected_output_contains=(
                    "'prefect-agent' typed work pools work with Prefect Agents instead of"
                    " Workers. Please use the 'prefect agent start' to start a Prefect Agent."
                ),
            )
```

Reply:
Oh okay this make sense . Thank you for helping out I am not very good at writing test :(

Reply:
@desertaxle I wanted to ask, will this function not contain a decorator like @pytest.mark.usefixtures("interactive_console")?

Reply:
No, this test shouldn't require an interactive console.

Reply:
Okay , I have made the changes you can review it now.

Reply:
@desertaxle One of the test is failing what could be the reason ?